lib/gris/cache_key.rb

```diff
@@ -0,0 +1,25 @@
+module Gris
+  module CacheKey
```

Comment:
We should add some coverage here. Ensuring that the `cache_key` and `self.cache_key_for` work as expected would be wise. I would add an explanation to the PR description about the decision to use this instead of Active Record's cache_key.

Likewise specs demonstrating how this module may be included on a model and expectations for how it should then behave seems important.